docs/reference/installation/_interfaces/kubectl.cue

```diff
@@ -0,0 +1,134 @@
+package metadata
+
+installation: _interfaces: kubectl: {
+	title:       "Kubectl"
+	description: """
+		The [Kubernetes command-line tool](\(urls.kubectl)), kubectl, allows
+		users to run commands against Kubernetes clusters facilitating in
+		application deployment, scaling, monitoring, and introspection.
+		"""
+
+	archs: ["x86_64", "ARM64"]
+	paths: {
+		bin:         null
+		bin_in_path: null
+		config:      null
+	}
+	platform_name: installation.platforms.kubernetes.name
+
+	roles: [Name=string]: {
+		commands: {
+			_resource_type: string
+			_role:          string
+			install:        "kubectl apply -k ."
+			logs:           #"kubectl logs -n vector \#(_resource_type)/vector-\#(_role)"#
+			reconfigure:    #"kubectl edit \#(_resource_type) vector-\#(_role)"#
```

Comment:
```suggestion
			reconfigure:    "kubectl apply -k ."
```

Reply:
The `reconfigure` command is about editing the Vector configuration, not applying. The `reload` command does that. Hence why I used `kubectl edit`.

Reply:
`kubectl edit` of the resource will edit the resource, and if there are any changes the rollout will be issued. In this case, the `reconfigure` command is `vim <config file>`, etc?